doc/source/whatsnew/v0.25.1.rst

```diff
@@ -129,7 +129,7 @@ Reshaping
 ^^^^^^^^^
 
 - A ``KeyError`` is now raised if ``.unstack()`` is called on a :class:`Series` or :class:`DataFrame` with a flat :class:`Index` passing a name which is not the correct one (:issue:`18303`)
--
+-  Bug in :meth:`DataFrame.crosstab` when margins set to True and normalize is not False, an error is raised. (:issue:`27500`)
```

Comment:
use double back-ticks on True, False and margins

Reply:
done!